notebooks/1_BayesianWorkflow/3_Ins_EndtoEndWorkflow.ipynb

```diff
@@ -0,0 +1,178 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1.3 End to End Statistics workflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Wikipedia  \n",
+    "_Statistics is a branch of mathematics dealing with data collection, organization, analysis, interpretation and presentation._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayesian Versus ...\n",
+    "It should be noted that Bayesian Statistics is one methodology in the broad field of statistics/machine learning. Below we briefly summarize the differences between Bayesian Statistics and other commonly known Statistics/Machine Learning methodologies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Bayesian statistics versus frequentists\n",
```

Comment:
Maybe you can add here, that frequentist have many estimators, while in Bayesian statistics we always compute posterior distribution and then we do whatever we want/need with that